classes/patterns/class-block-pattern.php

```diff
@@ -43,6 +43,8 @@ public static function get_list(): array {
 			BlankPage::class,
 			DeepDive::class,
 			GetInformed::class,
+			HomePage::class,
+			SideImageWithTextAndCta::class,
```

Comment:
Looks like you have this one declared twice 😬 Probably something went wrong while rebasing at some point!

Reply:
Good catch! I've just removed.